sdk/mixedreality/Azure.MixedReality.Authentication/src/Azure.MixedReality.Authentication.csproj

```diff
@@ -11,10 +11,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Core" />
     <PackageReference Include="System.IdentityModel.Tokens.Jwt" />
     <PackageReference Include="System.Text.Json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Azure.Core" />
+  </ItemGroup>
```

Comment:
double package reference. this seems unintended?

Reply:
Yep, accident.